perun-core/src/test/java/cz/metacentrum/perun/core/entry/VosManagerEntryIntegrationTest.java

```diff
@@ -74,6 +75,17 @@ public void createVo() throws Exception {
 		assertTrue("id must be greater than zero", newVo.getId() > 0);
 	}
 
+	@Test
+	public void createVoIsNotTenant() throws Exception {
+		System.out.println(CLASS_NAME + "createVoIsNotTenant");
```

Comment:
This behaviour is quite tricky, because it doesn't even raise an exception and just ignores the isTenant==true parameter. I'd say we should either allow this variant and set it to be tenant directly when creating the vo, or at least raise an exception.

Reply:
This actually follows the method's documentation in the RPC API that only VO's name and shortname are used and other parameters are ignored.  Therefore, I think an exception doesn't need to be thrown, but at least Javadoc of the method in other layers should be updated according to RPC API. And we discussed that we don't need the possibility to create tenants directly because it is a very rare use case and it would be probably an unnecessary breaking change. What do you think?

Reply:
We should remove the option from the constructor then.